xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/transactions/Transaction.java

```diff
@@ -84,28 +84,6 @@ public interface Transaction {
       .put(ImmutableUnlModify.class, TransactionType.UNL_MODIFY)
       .build();
 
-  /**
-   * Computes the fee necessary for a multisigned transaction.
-   *
-   * <p>The transaction cost of a multisigned transaction must be at least {@code (N + 1) * (the normal
-   * transaction cost)}, where {@code N} is the number of signatures provided.
-   *
-   * @param currentLedgerFeeDrops The current ledger fee, represented as an {@link XrpCurrencyAmount}.
-   * @param signerList            The {@link SignerListObject} containing the signers of the transaction.
-   *
-   * @return An {@link XrpCurrencyAmount} representing the multisig fee.
-   */
-  static XrpCurrencyAmount computeMultiSigFee(
-    final XrpCurrencyAmount currentLedgerFeeDrops,
-    final SignerListObject signerList
-  ) {
-    Objects.requireNonNull(currentLedgerFeeDrops);
-    Objects.requireNonNull(signerList);
-
-    return currentLedgerFeeDrops
-      .times(XrpCurrencyAmount.of(UnsignedLong.valueOf(signerList.signerEntries().size() + 1)));
-  }
```

Comment:
This needs to stay here until we release v3, otherwise this is a breaking change. Let's `@Deprecated` this instead